lib/rules/new-cap.js

```diff
@@ -160,6 +164,27 @@ module.exports = function(context) {
     }
 
     /**
+     * Check if capitalization allowed for a CallExpression member methods e.g Immutable.Map
+     * @param  {Object}  allowedMap Object mapping calleName to a Boolean
+     * @param {ASTNode} node CallExpression node
+     * @returns {Boolean} Returns true if the callee.calleName may be capitalized
+     */
+    function isCapAllowedByWildcard(allowedMap, node) {
+        // https://github.com/eslint/eslint/issues/5187
+        if (context.getSource(node.callee).indexOf(".") > 0) {
```

Comment:
Your calling getSource(node.callee) twice in this function, can you store in a variable instead?